modules/games.py

```diff
@@ -1663,6 +1664,11 @@ async def win(self, ctx, winning_game: PolyGame = None, *, winning_side_name: st
         else:
             utilities.unlock_game(winning_game.id)
             if confirm_win:
+                if data := winning_game.is_season_game():
+                    pro_value = 0 if data[1] == 'J' else 1
```

Comment:
Could this not just be 
```suggestion
                    pro_value = data[1] == 'P'
```

Reply:
No, as it's indexed via ints when it's read.